rainbow-me/rainbow
# Make iPhone 14 layouts happy (& migrate away from deprecated inset packages)

Fixes https://rainbowhaus.slack.com/archives/C02C2FVC6N6/p1663289973514509

## What changed (plus any additional context for devs)

This PR aims to migrate from the deprecated inset packages (`react-native-iphone-x-helper` & `react-native-safe-area-view`) to `react-native-safe-area-context`. As a result, Rainbow is now happy on the iPhone 14.

## Screen recordings / screenshots

https://www.loom.com/share/2576a20087bb4f328d8cf398fd97d177


## What to test

- Ensure that the sheets do not bleed above the iPhone 14 dynamic island,
- Ensure that the sheets still do not bleed for iPhone 10+ notches,
- Ensure that the sheets still look fine for Android devices & iPhones w/o notches.
